app/services/fan_out_on_write_service.rb

```diff
@@ -30,15 +31,25 @@ def call(status)
 
   def deliver_to_self(status)
     Rails.logger.debug "Delivering status #{status.id} to author"
-    FeedManager.instance.push(:home, status.account, status)
+    FeedManager.instance.push_to_home(status.account, status)
```

Comment:
What if you're in one of your own lists?

Reply:
You can't follow yourself so you can't add yourself to a list.